library/src-bootstrapped/scala/quoted/util/ExprMap.scala

```diff
@@ -0,0 +1,159 @@
+package scala.quoted.util
+
+import scala.quoted._
+
+trait ExprMap {
+
+  /** Map an expression `e` with a type `tpe` */
+  def transform[T](e: Expr[T])(given qctx: QuoteContext, tpe: Type[T]): Expr[T]
+
+  /** Map subexpressions an expression `e` with a type `tpe` */
+  def transformChildren[T](e: Expr[T])(given qctx: QuoteContext, tpe: Type[T]): Expr[T] = {
+    import qctx.tasty.{_, given}
+    class MapChildren() {
+
+      def transformStatement(tree: Statement)(given ctx: Context): Statement = {
```

Comment:
All the transforms on trees can be made `private`?

Reply:
Done